app/views/admin/content_pages/preview_of_live.erb

```diff
@@ -5,8 +5,18 @@
 <h1 class="govuk-heading-l">
   <%= @content_page.title %>
 </h1>
-<%= translate_markdown(@content_page.markdown) %>
-
+<% if @content_page.content_list %>
+<p class="govuk-body govuk-!-margin-bottom-1"> Contents </p>
+<% end %>
+<%= translate_markdown(@content_page.content_list) %>
+<% if @content_page.content_list %>
+<div class="gem-c-print-link govuk-!-margin-top-7" id="print-button">
+  <button class="govuk-link gem-c-print-link__button" onclick="window.print()" data-module="print-link" >Print this page</button>
+</div>
```

Comment:
I think the print button should be wrapped in a helper:
```ruby
def print_button(*additional_classes)
  button = '<button class="govuk-link gem-c-print-link__button" onclick="window.print()" data-module="print-link" >Print this page</button>'.html_safe
  classes = ["gem-c-print-link", "print-button"] + additional_classes
  content_tag :div, button, class: classes
end
```
Then here:
```ruby
<%= print_button("govuk-!-margin-top-7") %>
```
And in `app/views/admin/content_page_versions/preview_of_draft.erb`
```ruby
<%= print_button("govuk-!-margin-top-7", "govuk-!-margin-bottom-2") %>
```
Note - that I've put "print-button" as a class rather than the id, as I think there are use cases where there will be more than one print button on a page (at the top and bottom of long content for example)